packages/block-template-react/src/app.tsx

```diff
@@ -2,41 +2,101 @@ import {
   BlockComponent,
   useGraphBlockService,
 } from "@blockprotocol/graph/react";
-import { useCallback, useRef } from "react";
+import { useRef } from "react";
 
+/**
+ * The file referenced here provides some base styling for your block.
+ * You can delete it and write your own, but we encourage you to keep the styling scoped, e.g.
+ * - use CSS modules (this approach), e.g. overwrite the contents of .block in base.module.scss
+ * - use a CSS-in-JS solution
+ * - use a shadow DOM
+ * any of these ensure that your styling does not affect anything outside your block.
+ */
 import styles from "./base.module.scss";
 
+/**
+ * This defines the properties of the entity your block expects to be sent.
+ * This entity is available to your block on props.graph.blockEntity.
+ * To change the structure of the entity your block expects, change this type.
+ */
 type BlockEntityProperties = {
   name: string;
 };
 
+/**
+ * This function is to help illustrate a property being change when the button is pressed.
+ * You can remove it when you remove the button below.
+ */
```

Comment:
Not sure if we need to explain what this function does (it looks too obvious to me). Or maybe we can remove the second line. Imagine default React project explains us "you can remove this image import when you remove the `img` below" on the default React project. I felt like we're explaining "you should remove unused functions from you code" to developers. 

Maybe I'm wrong, just explained my gut feeling when I first saw this comment

Reply:
Thanks! Removed the second line in https://github.com/blockprotocol/blockprotocol/pull/598/commits/3c41770f16d43573e718be49b1e379a9d3444454